src/terms.jl

```diff
@@ -0,0 +1,296 @@
+import Base.+, Base.*, Base.length, Base.product
+
+# Abstract base class and definitions
+abstract Term
+
+function get_terms(term::Term)
+    return tuple(term)
+end
+
+function get_real_coefficients(term::Term)
+    return (Array(Float64, 0), Array(Float64, 0))
```

Comment:
That's correct. That's the default behavior for the abstract base class. If you look below you'll see the overloads for other types.